sormas-api/src/main/resources/strings.properties

```diff
@@ -991,7 +1000,10 @@ infoNoAccessToPersonEntities = You don't have the necessary user rights to view
 infoDashboardFinalLaboratoryResult = When a case has multiple samples, only the final laboratory result of the sample with the latest sample collection date is considered.
 infoBulkProcess = %d selected entries are currently being processed. This may take a while. The progress will be updated for each 20 entries that have been processed.
 infoBulkProcessFinished = <b>All selected entries have been processed!</b><br/>You can now close this window.
-infoBulkProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction or already archived.
+infoBulkProcessFinishedWithIneligibleItems = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, because they were not eligible.
+infoBulkDeleteProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction.
+infoBulkProcessFinishedWithSkips = <b>Bulk process has been successfully completed!</b><br/>However, some of the selected entries could not be processed, most likely because they were outside your jurisdiction, already archived or not eligible.
+infoBulkProcessNoEligibleEntries = <b>Bulk process has been cancelled because there are no eligible entries for this operation!
```

Comment:
See above

Reply:
A part of the messages, I've explained above, related to the infoBulkProcessNoEligibleEntries string:
Is used here (when > 40 entitiesare selected but none of the selected entities are eligible):
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/8941a7a6-b09c-4552-94e6-b0891513c97d)

It looks like this now (the reason why is not eligible can be handled here https://github.com/SORMAS-Foundation/SORMAS-Project/issues/12257):
![image](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/47103965/9e562b85-9b36-463a-97c8-576e648311ba)

Reply:
Fine by me :)